.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: csharp
```

Comment:
@pranavkm do you have the ability to enable Travis for this repo? It would be good to enable so we can ensure the functional tests pass on Linux